Wiki/BlankConfig/index.md

```diff
@@ -0,0 +1,44 @@
+---
+layout : wiki
+title : BlankConfig
+type : config
+weight : 500
+description : |
+  This is a description of how to configure Fvwm3 from scratch with a blank config file
+---
+
+* TOC
+{:toc}
+
+# Starting Fvwm3 with a Blank Config File
+
+Fvwm has been written with literally hundreds of options for most (if not all) aspects of window management.
+
+In order for windows to appear in the first place, there needs to be a root area (which, in turn, needs to "look like something"); a window needs to either have decorations or no decorations; if it has decorations, they need to be defined and styled; once a window is opened, it needs to open with some kind of geometry somewhere over the root area, etc...
+
+Imagine you need to define all these options _before_ you can make use of your window manager -- you'd be configuring for quite some time. To provide a working Fvwm3, all the necessary options have a default setting so that you can fire up Fvwm3 from the console and begin using it as your window management tool.
+
+Since Fvwm 2.6.7, "however", there has been a default configuration that ships together with a new Fvwm install, and when you run Fvwm3, it will be read from its default location (in most cases /usr/local/share/fvwm3/default-config/config) until you as the user direct Fvwm3 to another specific config file. 
+
+You could copy the default config to your own $HOME and start making Fvwm your own by adapting it (this option is described in greater detail in: [DefaultConfig](/DefaultConfig)).
+
+Or, and that is the fun thing about Fvwm3, you can start Fvwm3 with a blank, empty config file and start configuring from scratch without all the suggestions from either the default config or any other ready-made config file you might get from the internet ([fvwmforums.org/c/fvwm-themes/user-configurations](https://fvwmforums.org/c/fvwm-themes/user-configurations/)/, e.g.).
```

Comment:
Yeah -- linking to the user-configuration forums section is fine, but the new main fvwm3 page will have references to things like fvwm-{nightshade,crystal}, etc.  Do we want to refer to that instead?

Reply:
As the forum section is quite empty right now, I guess, a reference to the carousel members is a very good idea, indeed. 
Maybe keep the link to the forum as a second idea? I have learnt a lot about how to make Fvwm2 my own by reading other people's configs, and I have never warmed to using an fvwm-{XYZ}.

Reply:
Added the link to https:www.fvwm.org/index.html#configuring-fvwm , which I hope will be the anchor?